codegen/Cargo.toml

```diff
@@ -12,19 +12,32 @@ documentation = "https://docs.rs/subxt-codegen"
 homepage.workspace = true
 description = "Generate an API for interacting with a substrate node from FRAME metadata"
 
+[features]
+default = ["fetch_metadata"]
+fetch_metadata = ["dep:jsonrpsee", "dep:tokio", "dep:frame-metadata"]
+web = ["getrandom/js"]
```

Comment:
Is the `web` feature compatible with `fetch_metadata`?

Reply:
Ah, no it isn't. I'm gonna make sure we get a compile error when they are combined.

Reply:
can you explain why `getrandom` is needed? It's not clear to me why it's needed :P

Reply:
I'm not 100% if it's definitely needed here; @tadeohepperle now that the `fetch-metadata` feature flag is a thing, could you confirm whether we need to care about the `getrandom/js` feature?

Looking at `cargo tree -i getrandom` I think it's only poulled in when jsonrpsee is enabled anyway, so we shouldn't need to care otherwise :)

Reply:
@jsdw @niklasad1 I removed the `getrandom` dependency. Originally I wanted to have jsonrpsee available and needed this `getrandom/js` feature in the WASM, but later figured it might be unnecessary.